test/functional/api/vim_spec.lua

```diff
@@ -577,7 +577,6 @@ describe('API', function()
     local start_dir
 
     before_each(function()
-      clear()
```

Comment:
should probably have the `nvim_set_current_dir` tests in a single `it()` instead of having multiple `it()` blocks that depend on each toher.

Reply:
They don't depend on each other, because there is still a `before_each(clear)` at the top.